ui/src/index.tsx

```diff
@@ -20,13 +24,21 @@ export const MyContext = React.createContext<IAppContext>(null);
 const AppProvider = (props) => {
   const [store, setStore] = useState({
     volume: null,
+    version: "",
   });
 
   const actions = {
     setVolume: (value: IVolumeRow | null) =>
       setStore((oldStore) => ({ ...oldStore, volume: value })),
   };
 
+  useEffect(() => {
+    ddClient.docker.cli.exec("extension version", []).then((output) => {
```

Comment:
If this command were to fail, I understand that the push/pull to volume functionality would be hidden by default, right?

Reply:
Yes, if the sdkVersion is not retrieved, it will be an empty string, and the comparison will return `false`. I added a catch block to make it explicit.